clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/MetricUpdater.java

```diff
@@ -75,14 +75,8 @@ public void updateMasterElectionMetrics(Map<Integer, Integer> data) {
         metricReporter.set("agreed-master-votes", maxCount);
     }
 
-    public void becameMaster() {
-        metricReporter.set("is-master", 1);
-        metricReporter.add("master-change", 1);
-    }
-
-    public void noLongerMaster() {
-        metricReporter.set("is-master", 0);
-        metricReporter.add("master-change", 1);
+    public void isMaster(boolean isMaster) {
```

Comment:
Consider renaming to something like `updateMasterState` to make the method sound less like a getter